BUIP-HF/replay-protected-sighash.md

```diff
@@ -0,0 +1,191 @@
+# BUIP-HF Digest for replay protected signature verification accross hard forks
+
+## Abstract
+
+This document describes proposed requirements and design for a reusable signing mechanism ensuring replay protection in the event of a hard fork. It provides a way for users to create transactions which are only valid on one branch of the fork.
```

Comment:
<strike>only valid on one branch of the fork</strike>
invalid on forks lacking support for the mechanism and a fork-specific ID